pkgs/top-level/all-packages.nix

```diff
@@ -18786,6 +18786,13 @@ with pkgs;
 
   sublime3-dev = sublime3Packages.sublime3-dev;
 
+  sublimeMergePackages = recurseIntoAttrs (callPackage ../development/tools/sublime-merge/packages.nix { });
+
+  sublimeMerge = sublimeMergePackages.sublimeMerge;
```

Comment:
You can do `inherit (sublimeMergePackages) sublimeMerge sublimeMerge-dev;`

Reply:
Oh, that is much nicer.